### DISCUSSION TITLE
When to use t.get(...) inside tTestWidgets and when not

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#352] [Q&A] [answered]
Sorry, this might be a silly question but I do not get it (I am still new to Flutter). Can you explain when to use ` await t.get(find. ...)` inside `tTestWidgets` and when not? E.g. both statements in the sample below work flawlessly. I do not understand when it is necessary to use t.get
```
group('simple test group', () {
      tTestWidgets('simple test 1', (t) async {

        //both commands work just fine - why prefer the onee over the other?
        await t.get(find.text('blah')).should(findsOneWidget);
        await find.text('blah').should(findsOneWidget);

        //both commands work just fine - why prefer the onee over the other?
        await find.text('Submit').tap();
        await t.get(find.text('Submit')).tap();
      });
    });
  });
```

**Comment by fzyzcjy** [comment, marked as answer]
Yes, the code without t.get is a quick syntax sugar I have added later. So feel free to use the short version (and fallback to the long version when something is too complex and does not provide such syntax sugar)

**Reply by lehmannk** [reply]
Thanks for the explanation

**Reply by fzyzcjy** [reply]
You are welcome!
